runhouse/rns/folders/folder.py

```diff
@@ -208,15 +224,17 @@ def data_config(self, data_config):
     @property
     def _fs_str(self):
         if isinstance(self.system, Resource):  # if system is a cluster
-            # TODO [DG] Return 'file' if we're on this cluster
+            if self.system.rns_address == _current_cluster("rns_address"):
```

Comment:
Is there an easy way to add a test for this case? would require having a folder on a cluster and testing from within the cluster that it returns "file" and not "ssh"

Reply:
Yes, you can create a folder with its system set to a cluster, then define a function which accepts a folder object and returns it's _fs_str, then send that function to the cluster and call it on your folder object, you should get back "file" and not "ssh" (but get back "ssh" if you run the function locally). Does that make sense?